### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kushal34712**
- change the heading place of the readme.md file 

- added line after banner 

- fixed typo 

- corrected grammar

- added licence and its path so it can be directly accessible

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +56,7 @@ pip install trl
 
 ### From source
 
-If you want to use the latest features before an official release you can install from source:
+If you want to use the latest features before an official release you can install them from source:
```
Comment (qgallouedec) [inline README.md:59]: Here, you install the lib, not the features

**Comment by kushal34712** [comment]
@qgallouedec made the changes requested

File: README.md [hunk 2]
```diff
@@ -23,7 +25,7 @@
 
 ## What is it?
 
-TRL is a library that post-trains LLMs and diffusion models using methods such as Supervised Fine-Tuning (SFT), Proximal Policy Optimization (PPO), and Direct Preference Optimization (DPO). 
+TRL is a library that post-trains LLMs and diffusion models using methods such as Supervised fine tuning (SFT), Proximal Policy Optimization (PPO), and Direct Preference Optimization (DPO).
```
Comment (qgallouedec) [inline README.md:28]: I'd revert this one

File: README.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
+# TRL - Transformer Reinforcement Learning
```
Comment (qgallouedec) [inline README.md:1]: any motivation to move the banner down?

File: README.md [hunk 4]
```diff
@@ -51,7 +56,7 @@ pip install trl
 
 ### From source
 
-If you want to use the latest features before an official release, you can install TRL from source:
+If you want to use the latest features before an official release, you can install TRL from the source:
```
Comment (qgallouedec) [inline README.md:59]:
```suggestion
If you want to use the latest features before an official release, you can install TRL from source:
```

**Comment by qgallouedec** [comment]
> @qgallouedec made the changes requested

thanks, I let @lewtun and @edbeeching reviewing this one

File: README.md (hunk 3 above)
Comment (kushal34712) [reply README.md:1]: Like added this as a heading

File: README.md [hunk 5]
```diff
@@ -32,11 +34,14 @@ The library is built on top of [🤗 Transformers](https://github.com/huggingfac
 
 - **`Efficient and scalable`**: 
     - [🤗 Accelerate](https://github.com/huggingface/accelerate) is the backbone of TRL that models training to scale from a single GPU to a large-scale multi-node cluster with methods such as DDP and DeepSpeed.
-    - [`PEFT`](https://github.com/huggingface/peft) is fully integrated and allows to train even the largest models on modest hardware with quantization and methods such as LoRA or QLoRA.
+    - [`PEFT`](https://github.com/huggingface/peft) is fully integrated and allows to training of even the largest models on modest hardware with quantization and methods such as LoRA or QLoRA.
```
Comment (edbeeching) [inline README.md:37]:
```suggestion
    - [`PEFT`](https://github.com/huggingface/peft) is fully integrated and allows users to train even the largest models on modest hardware with quantization and methods such as LoRA or QLoRA.
```

File: README.md (hunk 4 above)
Comment (edbeeching) [reply README.md:59]: +1

**Review by edbeeching — APPROVED** [review]
LGTM barring minor comments / reverts

File: README.md (hunk 2 above)
Comment (qgallouedec) [reply README.md:28]:
```suggestion
TRL is a library that post-trains LLMs and diffusion models using methods such as Supervised Fine-Tuning (SFT), Proximal Policy Optimization (PPO), and Direct Preference Optimization (DPO).
```

**Pull request merged**